Podcast/EpisodeSubjectView.swift

```diff
@@ -122,6 +122,7 @@ class EpisodeSubjectView: UIView {
         
         descriptionLabel.snp.makeConstraints { make in
             make.top.greaterThanOrEqualTo(podcastImage.snp.bottom).offset(descriptionLabelOffset)
+            make.top.greaterThanOrEqualTo(dateTimeLabel.snp.bottom).offset(descriptionLabelOffset)
```

Comment:
ok the constraint errors are coming from this line... but making the priority 999 doesn't work properly either because then the episode cells look like this:
![image](https://user-images.githubusercontent.com/20246620/36743045-3cbc746c-1bb7-11e8-9a94-db3dfee6d6e3.png)

so i think the constraint has to change depending on whether or not the `dateTimeLabel` exceeds one line, if it doesn't the offset should just be `make.top.greaterThanOrEqualTo(podcastImage.snp.bottom).offset(descriptionLabelOffset)`